server/validations/clinicrate.js

```diff
@@ -0,0 +1,24 @@
+import validator from "validator";
+import { isEmpty, isRequired } from "./is-empty";
+
+export const validateClinicRateFields = data => {
+  let errors = {};
+
+  data.user = isRequired(data.user);
+  data.value = isRequired(data.value);
+
+  if (Validator.isEmpty(data.clinic)) {
+    errors.clinic = "clinic field is required";
+  }
+  if (Validator.isEmpty(data.user)) {
+    errors.user = "user field is required";
+  }
+  if (Validator.isEmpty(data.review.username)) {
+    errors.review = "username field is required";
```

Comment:
clinicrate does not contain a property named review, it have a numerical value calle value